db/plugins.py

```diff
@@ -555,12 +555,12 @@ def get_search_index():
 
     # Don't show plugin managers because they're not technically plugins, and
     # also our usage counts for them are not all accurate.
-    query = query.filter(r.all(
-        r.row['slug'] != 'vundle',
-        r.row['slug'] != 'neobundle-vim',
-        r.row['slug'] != 'neobundle-vim-back-to-december',
-        r.row['slug'] != 'pathogen-vim',
-    ))
+    query = query.filter(
+        (r.row['slug'] != 'vundle') &
+        (r.row['slug'] != 'neobundle-vim') &
+        (r.row['slug'] != 'neobundle-vim-back-to-december') &
+        (r.row['slug'] != 'pathogen-vim')
+    )
```

Comment:
I'm not super familiar with the Python syntax, but you could rewrite this with `contains`. Something like: 

```
query = query.filter(
  lambda row: r.expr(['vundle', 'neobundle-vim', 'neobundle-vim-back-to-december', 'pathogen-vim']).contains(r.row['slug']).not()
)
```
Then you could move out that array out of the query:
```
unallowed_slugs = ['vundle', 'neobundle-vim', 'neobundle-vim-back-to-december', 'pathogen-vim']
query = query.filter(
  lambda row: r.expr(slugs).contains(r.row['slug']).not()
)
```